data_registry/models.py

```diff
@@ -300,13 +301,20 @@ class UpdateFrequency(models.TextChoices):
     frozen = models.BooleanField(
         default=False, help_text="If the spider is broken, check this box to prevent the scheduling of new jobs."
     )
+    no_data_rationale = models.TextField(
+        blank=True,
+        default="",
+        verbose_name="no data rationale",
+        help_text="A short rationale of why an existing publication does not have data in the Data Registry. If this"
+        "field is set, then the collection should be frozen and their retrieval frequency set to NEVER",
```

Comment:
Frozen is enough. Retrieval frequency = Never means "This dataset is no longer updated by the publisher".